packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -61,19 +71,51 @@
      */
     @Prop({ default: () => NoElement })
     public animation!: string | typeof Vue;
```

Comment:
I imagine that it would be a breaking change and we prefer to avoid that issue but now that we have three different animation variables (`animation`, `hoverAnimation`, `imageAnimation`) I suggest a refactor on namings, something like:
1. `loadAnimation`, the prop to specify how to animate between placeholder & result image & fallback image,
2. `hoverAnimation`, the prop to specify how to change between images when hovering the result.
3. `animation` or `currentAnimation`, the getter that calculates the specific animation to apply at each moment.